tests/test-run.sh

```diff
@@ -226,7 +226,7 @@ fi
 # Test --die-with-parent
 
 cat >lockf-n.py <<EOF
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
`centos7` build now fails:

```
+ ./lockf-n.py ./lock nowait
/usr/bin/env: python3: No such file or directory
```

The `ci/papr.sh` probably now needs to `yum -y install ... python3` or similar.

Reply:
Please change `ci/papr.sh` to add `python3` to the list of packages on the `yum` command-line, something like this:

```diff
     yum -y install git autoconf automake libtool make gcc redhat-rpm-config \
         libcap-devel  'pkgconfig(libselinux)' 'libxslt' 'docbook-style-xsl' \
-        lib{a,ub,t}san /usr/bin/eu-readelf rsync
+        lib{a,ub,t}san /usr/bin/eu-readelf rsync python3
```

That should hopefully be enough to fix the CI failure.

(You'll be doing this blindly - I don't know how to run the CI myself - but it's a simple enough change to make that way with reasonable confidence.)